README.md

```diff
@@ -59,6 +59,18 @@ Returns:
 We are thrilled you are considering contributing!
 Please read our [guidelines for contributing](CONTRIBUTING.md).
 
+## Creating new translator modules
```

Comment:
I'm thinking the Contributing bit should remain right above the Licensing bit.

Reply:
Sure